### ISSUE TITLE
Viewing a PipelineFamily shows Pipelines outside that family

### ISSUE THREAD

**Issue by rhliang** [cfe-lab/Kive#699]
Just noticed this on my workstation.  This is what I see when I look at the Sums and Products PipelineFamily:

![pipelinefamilyshowsotherpipelines](https://user-images.githubusercontent.com/5862525/28486611-8634d4ea-6e38-11e7-8d12-e202278c5568.png)

This is what the server outputs when I do it:
```
[21/Jul/2017 17:15:30] "GET /api/pipelines/?is_granted=true&filters%5B0%5D%5Bkey%5D=pipelinefamily_id&filters%5B0%5D%5Bval%5D=1&page_size=25&page=1 HTTP/1.1" 200 13079
```

I went into the shell and confirmed that the MiCall demo pipeline does not in fact belong to this PipelineFamily, the backend looks coherent.

**Comment by donkirkby** [comment]
Related to the changes for issue #659?

**Comment by rhliang** [comment]
Yep, looks like the `filter_queryset` method of the PipelineViewSet and PipelineFamilyViewSet got removed.  I put them back in and it works again.

**Comment by donkirkby** [comment]
Does the fix for #659 still work?

**Comment by rhliang** [comment]
Shoot, no it doesn't, I better dig back into that.
